service-users-specs/src/test/resources/scenarios/userCreation.feature

```diff
@@ -0,0 +1,35 @@
+Feature: user creation
+
+  Background:
+    Given there is a User and Team Server
+
+  Scenario: create a user
+    Given I have a user payload
+    When I POST it to the /users endpoint
+    Then I receive a 201 status code
+
+  Scenario: cannot create twice the same user
+    Given I have a user named Johny payload
+    When I POST it to the /users endpoint
+    Then I receive a 201 status code
+    When I POST it to the /users endpoint
+    Then I receive a 403 status code
+
+  Scenario: create two users with capitalized username
+    Given I have a user named George payload
+    When I POST it to the /users endpoint
+    Then I receive a 201 status code
+    Given I have a user named george payload
+    When I POST it to the /users endpoint
+    Then I receive a 403 status code
+
+  Scenario: user endpoint given after creation
+    Given I have a user named Alfred payload
+    When I POST it to the /users endpoint
+    Then I receive a 201 status code
+    Then I receive an endpoint to the user
```

Comment:
Would be interesting to do a GET to this URL and check the payload.

Reply:
done in b36b4fc